index.Rmd

```diff
@@ -0,0 +1,181 @@
+---
+title: "The Mother's Age and Birth Rates in the US"
+author: "Helen Chamley"
+date: '2022-05-09'
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = 'center')
+```
+
+```{r}
+source(here::here("scripts", "functions.R"))
+source(here::here("scripts", "get_data.R"))
+```
+
+```{r}
+births_clean <-
+  births %>% 
+    slice(-c(2,4:55, 57:108, 110:161, 163:214, 216:267, 
```

Comment:
I feel like there must be a better way to do this that would more readily make it clear what is happening. This feels ripe for error. Doing something like this would be much more legible.:

```
selected_rows <- c("Births (People)", "Less than 15 (People)", ... , "Some college credit, but not a degree (People)")

births %>%
   rename("Type" = Year) %>%
   filter(Type %in% selected_rows)
```